src/api/api.py

```
@@ -0,0 +1,112 @@
+        # 2. Создаем UseCase и запускаем в фоне
+        save_use_case = SaveDataUseCase(loader, repo)
+        
+        # Передаем в execute. Если это телеграм, source_path — имя чата.
+        background_tasks.add_task(save_use_case.execute, request.source_path)
+
+        return {
+            "status": "accepted", 
+            "message": f"Загрузка из {request.source_type} запущена в фоновом режиме"
+        }
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)